.github/workflows/test-e2e-cli.yml

```
@@ -6,12 +6,18 @@ on:
       version:
         type: string
         required: false
+      without-cache:
+        type: boolean
+        default: false
+        required: false
```

```suggestion
        required: false
        default: false
```
Swap so we know that default is at the end as there both variables are the same